string path
uint64 size
uint64 free
float64 used
isaac_ros_nova_recorder/RosbagInfo[] rosbags
